__tests__/app.test.js

```diff
@@ -8,6 +9,22 @@ beforeEach(() => seed(data))
 
 afterAll(() => db.end())
 
+describe("/api", () => {
+    test("GET 200, returns object describing all endpoints", () => {        
+        return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({ body: { endpoints } }) => {
+            return Promise.all([endpoints, fs.readFile(`${__dirname}/../endpoints.json`, "utf-8")])
+        })
+        .then(([endpointsOutput, jsonExpectedEndpoints]) => {
+            const expectedEndpoints = JSON.parse(jsonExpectedEndpoints) 
```

Comment:
Please see Rose's post in the main student channel on a better in which to test this endpoint